2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/metrics/StaticMetrics.scala

```diff
@@ -0,0 +1,21 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream.metrics
+
+import java.io.File
+
+import scala.util.Try
+
+object StaticMetrics {
+
+  def javaVersion: String = classOf[Runtime].getPackage.getImplementationVersion
+
+  def scalaVersion: String = Try {
+    val props = new java.util.Properties
+    props.load(getClass.getResourceAsStream("/library.properties"))
+    props.getProperty("version.number")
+  }.getOrElse("")
+
+  def runnableName: String = Try {
+    new File(getClass.getProtectionDomain.getCodeSource.getLocation.getPath).getName
+  }.getOrElse("")
```

Comment:
this can just be https://github.com/snowplow/snowplow/blob/08d2f51b44148169427e5f423f7ea4dd489af9e5/2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala#L73-L74 no need for the jar name

Reply:
Thank you for the hint. I have left only the version, as it is only one `/metrics` per service and the name is not necessary